README.txt

```diff
@@ -160,3 +160,9 @@ Automatic updates should work smoothly, but we still recommend you back up your
 **ArkPay**
 
 * Update - Code/Functions update.
+
+= 1.0.11 2024-04-23 =
+
+**ArkPay**
+
+* Update - Code/Functions update.
```

Comment:
```suggestion
* Update - Code/Functions update.
```

Not a meaningful description, add something that explains why the code has been updated